visualization/FairnessDashboard/src/Controls/ModelComparisonChart.tsx

```diff
@@ -154,24 +154,25 @@ export class ModelComparisonChart extends React.PureComponent<IModelComparisonPr
             formattedMinDisparity,
             formattedMaxDisparity
         );
+        const metricTitleAppropriateCase = selectedMetric.alwaysUpperCase ? selectedMetric.title : selectedMetric.title.toLowerCase();
         const insights3 = localization.formatString(
             localization.ModelComparison.insightsText3,
-            selectedMetric.alwaysUpperCase ? selectedMetric.title : selectedMetric.title.toLowerCase(),
+            metricTitleAppropriateCase,
```

Comment:
ha, I was actually referring to line 185 and the two places its used, but this looks nice too! Its more of a nice to have for clean code, but it does look nice. Thanks Roman :)

Reply:
Ah I see! Added that as well.